WalletWasabi.Gui/Controls/WalletExplorer/SendTabViewModel.cs

```diff
@@ -891,6 +863,18 @@ private async Task DoDequeueAsync(IEnumerable<CoinViewModel> selectedCoins)
 			}
 		}
 
+		public string FeeText => FeeRate is null ? "" :
+										(FeeDisplayFormat == FeeDisplayFormat.SatoshiPerByte ? $"(~ {FeeRate.SatoshiPerByte} sat/byte)" :
+										 (FeeDisplayFormat == FeeDisplayFormat.USD ? $"(~ ${UsdFee.ToString("0.##")})" :
+										  (FeeDisplayFormat == FeeDisplayFormat.BTC ? $"(~ {EstimatedBtcFee.ToString(false, false)} BTC)" :
+										   (FeeDisplayFormat == FeeDisplayFormat.Percentage ? $"(~ {FeePercentage.ToString("0.#")} %)" : throw new NotSupportedException("This is impossible.")))));
```

Comment:
Can be written like this.
```
public string FeeText => FeeRate is null
	? ""
	: FeeDisplayFormat == FeeDisplayFormat.SatoshiPerByte
		? $"(~ {FeeRate.SatoshiPerByte} sat/byte)"
		: FeeDisplayFormat == FeeDisplayFormat.USD
			? $"(~ ${UsdFee.ToString("0.##")})"
			: FeeDisplayFormat == FeeDisplayFormat.BTC
				? $"(~ {EstimatedBtcFee.ToString(false, false)} BTC)"
				: FeeDisplayFormat == FeeDisplayFormat.Percentage
					? $"(~ {FeePercentage.ToString("0.#")} %)"
					: throw new NotSupportedException("This is impossible.");
```

Reply:
I think it is more readable how I have it.